index.html

```diff
@@ -60,104 +60,109 @@ <h1 class="name">I’m Nadiia.
                 <h2>Tonik</h2>
                 <ul class="spec liststyle">
-                  <li class="dark">CANOPY</li>
-                  <li class="light"><img src="Img/Counter.svg" alt=""></li>
-                  <li class="light">Back End Dev</li>
-                  <li class="light"><img src="Img/Counter.svg" alt=""></li>
-                  <li class="light">2015</li>
+                  <li class="specItem dark">CANOPY</li>
+                  <li class="specItem light"><img src="Img/Counter.svg" alt=""></li>
+                  <li class="specItem light">Back End Dev</li>
+                  <li class="specItem light"><img src="Img/Counter.svg" alt=""></li>
+                  <li class="specItem light">2015</li>
                 </ul>
                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 <ul class="lng liststyle">
-                  <li>html</li>
-                  <li>css</li>
-                  <li>javascript</li>
+                  <li class="tech">html</li>
+                  <li class="tech">css</li>
+                  <li class="tech">javascript</li>
                 </ul>
                 <button class="btn" type="button" name="details">See project</button>
               </div>
               </div>
             </div>
 
-            <div class="item4">
+            <div id="item4" class="item4 card">
               <div class="item reverse round">
               <a href="#"><img class="snapshot" src="Img/SnapshootPortfolio1.png" alt="project 4"></a>
               <div class="item-flex">
                 <h2>Tonik</h2>
                 <ul class="spec liststyle">
-                  <li class="dark">CANOPY</li>
-                  <li class="light"><img src="Img/Counter.svg" alt=""></li>
-                  <li class="light">Back End Dev</li>
-                  <li class="light"><img src="Img/Counter.svg" alt=""></li>
-                  <li class="light">2015</li>
+                  <li class="specItem dark">CANOPY</li>
+                  <li class="specItem light"><img src="Img/Counter.svg" alt=""></li>
+                  <li class="specItem light">Back End Dev</li>
+                  <li class="specItem light"><img src="Img/Counter.svg" alt=""></li>
+                  <li class="specItem light">2015</li>
                 </ul>
                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 <ul class="lng liststyle">
-                  <li>html</li>
-                  <li>css</li>
-                  <li>javascript</li>
+                  <li class="tech">html</li>
+                  <li class="tech">css</li>
+                  <li class="tech">javascript</li>
                 </ul>
                 <button class="btn" type="button" name="details">See project</button>
               </div>
               </div>
             </div>
-          </div>
 
+          </div>-->
           </section>
```

Comment:
- [ ] This section (Projects section) should be created dynamically with the information from the projects array, kindly remove this code from your HTML, and instead add it via JavaScript.